const.go

```diff
@@ -45,6 +45,8 @@ const (
 	clientSecureConn
 	clientMultiStatements
 	clientMultiResults
+	clientPSMultiResults
```

Comment:
clientPSMultiResults is not used by anything cleartext related but was added to use the clientPluginAuth capability flag. This may be a non issue now as I see you have updated const.go in master with the full range of capability flags.